### PR TITLE
PLATFORM-1962: Migrate Payment “app”

### PR DESCRIPTION
**Pull request by lilyfromseattle**
I fixed all type errors and tests pass, but for some reason the Payment story won't show up in storybooks. If anyone has any insight into why this might be please let me know.

https://artsyproduct.atlassian.net/browse/PLATFORM-1962

### CODE REVIEW

**Comment by mzikherman** [comment]
Can you close and re-open this PR, but instead of pointing to `master`, point to `mp-v2-migration`? That'll clean up the diff.

File: src/Components/Payment/SavedCreditCards.tsx [hunk 1]
```diff
@@ -151,6 +151,7 @@ export class CreditCard extends React.Component<
```
Comment (sweir27) [inline src/Components/Payment/SavedCreditCards.tsx]: I can't comment on the exact line, but it looks like you'll need to make some changes above (line 136/137 still reference `__id`)

Similarly, in the `deleteCreditCard` method you may need to update the `id`.

If you can pull up storybooks (or link to your local force) you should be able to test these cases!

Comment (lilyfromseattle) [reply src/Components/Payment/SavedCreditCards.tsx]: Whoops my bad

**Comment by sweir27** [comment]
Nice!!

**Comment by artsyit** [comment]
:rocket: PR was released in v22.0.0 :rocket:

**Pull request merged**